packages/run-npm/src/index.ts

```diff
@@ -3,10 +3,10 @@ import path from 'path'
 import spawn from 'cross-spawn'
 import PATH from 'path-name'
 
-export default function runNpm (npmPath: string | undefined, args: string[]) {
+export default function runNpm (npmPath: string | undefined, args: string[], cwd?: string) {
   const npm = npmPath ?? 'npm'
   return runScriptSync(npm, args, {
-    cwd: process.cwd(),
+    cwd: cwd || process.cwd(),
```

Comment:
probably

```suggestion
    cwd: cwd ?? process.cwd(),
```

Reply:
I don't think an empty string is ever going to be an ok cwd

Reply:
well, but the linter doesn't know that, I the end it shouldn't matter, I will change it to "??"